### PR TITLE
Add more information on force_mask

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@giuseppe @vrothberg @TomSweeneyRedHat @saschagrunert @nalind PTAL

File: docs/containers-storage.conf.5.md [hunk 1]
```diff
@@ -168,11 +168,38 @@ The `storage.options.overlay` table supports the following options:
...
 
 **force_mask** = "0000|shared|private"
-  force_mask sets an octal permission mask that is used for the new files and directories.  The values "native", "shared", "private" are also accepted.  (default: "native")
-  "private": it is equivalent to 0700.  The owner has rwx access to the files.
-  "shared": it is equivalent to 0755.  The owner has rwx access to the files and everyone else can read, access and execute them.
-  Note: The force_mask Flag is an experimental feature, it could change in the future.
-  When "force_mask" is used then the original permission mask is stored in the "user.containers.override_stat" xattr.
+  ForceMask specifies the permissions mask that is used for new files and
+directories.
+The values "native", "shared", "private" are accepted.  (default: "native").
```
Comment (giuseppe) [inline docs/containers-storage.conf.5.md:173]: that was a mistake in my version.  There is no "native" anymore.  The default is the empty string.

File: docs/containers-storage.conf.5.md [hunk 2]
```diff
@@ -168,11 +168,38 @@ The `storage.options.overlay` table supports the following options:
...
+Octal permission masks are also accepted.
+
+  `native`: Equivalent to 0000.
```
Comment (giuseppe) [inline docs/containers-storage.conf.5.md:176]: similarly here, "native" is not used.  The default value is the unset value and that means the permissions are left unchanged

**Review by giuseppe — COMMENTED** [review]
one comment, that was a leftover in my previous PR

File: docs/containers-storage.conf.5.md (hunk 2 above)
Comment (rhatdan) [reply docs/containers-storage.conf.5.md]: Removed "native"

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL

**Review by giuseppe — APPROVED** [review]
LGTM

File: docs/containers-storage.conf.5.md [hunk 3]
```diff
@@ -168,11 +168,37 @@ The `storage.options.overlay` table supports the following options:
...
 
 **force_mask** = "0000|shared|private"
-  force_mask sets an octal permission mask that is used for the new files and directories.  The values "native", "shared", "private" are also accepted.  (default: "native")
-  "private": it is equivalent to 0700.  The owner has rwx access to the files.
-  "shared": it is equivalent to 0755.  The owner has rwx access to the files and everyone else can read, access and execute them.
-  Note: The force_mask Flag is an experimental feature, it could change in the future.
-  When "force_mask" is used then the original permission mask is stored in the "user.containers.override_stat" xattr.
+  ForceMask specifies the permissions mask that is used for new files and
+directories.
+The values "shared", "private" are accepted.  (default: ""). Octal permission
```
Comment (TomSweeneyRedHat) [inline docs/containers-storage.conf.5.md:173]:
```suggestion
The values "shared" and "private" are accepted.  (default: ""). Octal permission
```

File: docs/containers-storage.conf.5.md [hunk 4]
```diff
@@ -168,11 +168,37 @@ The `storage.options.overlay` table supports the following options:
...
+masks are also accepted.
+
+  ``: Not set
+     All files/directories, get set with the permissions identified within the
+image.
+
+  `private`: it is equivalent to 0700.
+     All files/directories get set with 0700 permissions.  The owner has rwx
+access to the files. No other users on the system can access the files.
+This setting could be used with networked based homedirs.
```
Comment (TomSweeneyRedHat) [inline docs/containers-storage.conf.5.md:183]: homedirs or home directories?  I think probably home directories.

File: docs/containers-storage.conf.5.md [hunk 5]
```diff
@@ -168,11 +168,37 @@ The `storage.options.overlay` table supports the following options:
...
+
+  `shared`: it is equivalent to 0755.
+     The owner has rwx access to the files and everyone else can read, access
+and execute them. This setting is useful for sharing containers storage
+with other users.  For instance have a storage owned by root but shared
```
Comment (TomSweeneyRedHat) [inline docs/containers-storage.conf.5.md:188]:
```suggestion
with other users.  For instance, a storage owned by root could be shared
```

File: storage.conf [hunk 6]
```diff
@@ -82,8 +82,38 @@ mountopt = "nodev"
 # Size is used to set a maximum size of the container image.
 # size = ""
 
-# ForceMask specifies the permissions mask to use for created files.
-# force_mask = "native"
+# ForceMask specifies the permissions mask that is used for new files and
+# directories.
+#
+# The values "shared", "private" are accepted.
```
Comment (TomSweeneyRedHat) [inline storage.conf:88]:
```suggestion
# The values "shared" and "private" are accepted.
```

File: storage.conf [hunk 7]
```diff
@@ -82,8 +82,38 @@ mountopt = "nodev"
...
+# Octal permission masks are also accepted.
+#
+#  "": No value specified.
+#     All files/directories, get set with the permissions identified within the
+#     image.
+#  "private": it is equivalent to 0700.
+#     All files/directories get set with 0700 permissions.  The owner has rwx
+#     access to the files. No other users on the system can access the files.
+#     This setting could be used with networked based homedirs.
```
Comment (TomSweeneyRedHat) [inline storage.conf:97]: homdirs/home directories here too

File: storage.conf [hunk 8]
```diff
@@ -82,8 +82,38 @@ mountopt = "nodev"
...
+#  "shared": it is equivalent to 0755.
+#     The owner has rwx access to the files and everyone else can read, access
+#     and execute them. This setting is useful for sharing containers storage
+#     with other users.  For instance have a storage owned by root but shared
```
Comment (TomSweeneyRedHat) [inline storage.conf:101]: As above

**Review by TomSweeneyRedHat — COMMENTED** [review]
Just a couple nits, otherwise a nice addition.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM
thanks for picking those nits.

**Pull request merged**